eksctl/Makefile

```diff
@@ -23,10 +24,10 @@ create_node_group:
 		--name ${publicNgName} \
 		--cluster ${clusterName} \
 		--region ${region} \
-		--node-type t3a.small \
-		--nodes 3 \
+		--node-type t3.medium \
```

Comment:
I'm using a t3.medium now. 
They need a little headroom so that you can roll out deployments